### ISSUE TITLE
Some bugs in motion_tracking.py

### ISSUE THREAD

**Issue by OTT123** [LeCAR-Lab/ASAP#40]
# bug 1
In motion_tracking.py:

`from isaac_utils.rotations import (
    my_quat_rotate,
    quat_to_tan_norm, 
    calc_heading_quat_inv,
    calc_heading_quat,
    quat_mul,
    quat_conjugate,
    quat_to_angle_axis,
    quat_rotate_inverse,
    xyzw_to_wxyz,
    wxyz_to_xyzw
)`

but there's no quat_to_tan_norm defined in isaac_utils. 

## how to fix this
Comment this code
`from isaac_utils.rotations import (
    my_quat_rotate,
    # quat_to_tan_norm, 
    calc_heading_quat_inv,
    calc_heading_quat,
    quat_mul,
    quat_conjugate,
    quat_to_angle_axis,
    quat_rotate_inverse,
    xyzw_to_wxyz,
    wxyz_to_xyzw
)`

# bug 2
In motion_tracking.py:

`    def _reward_teleop_body_rotation_extend(self):
        rotation_diff = quat_to_angle_axis(self.dif_global_body_rot, w_last=True)[0]
        diff_body_rot_dist = (rotation_diff**2).mean(dim=-1)
        r_body_rot = torch.exp(-diff_body_rot_dist / self.config.rewards.reward_tracking_sigma.teleop_body_rot)
        return r_body_rot`

Where quat_to_angle_axis receives tow params, but in isaac_utils
`
def quat_to_angle_axis(q):
    # type: (Tensor) -> Tuple[Tensor, Tensor]
    # computes axis-angle representation from quaternion q
    # q must be normalized
    # ZL: could have issues. 
    min_theta = 1e-5
    qx, qy, qz, qw = 0, 1, 2, 3

    sin_theta = torch.sqrt(1 - q[..., qw] * q[..., qw])
    angle = 2 * torch.acos(q[..., qw])
    angle = normalize_angle(angle)
    sin_theta_expand = sin_theta.unsqueeze(-1)
    axis = q[..., qx:qw] / sin_theta_expand

    mask = torch.abs(sin_theta) > min_theta
    default_axis = torch.zeros_like(axis)
    default_axis[..., -1] = 1

    angle = torch.where(mask, angle, torch.zeros_like(angle))
    mask_expand = mask.unsqueeze(-1)
    axis = torch.where(mask_expand, axis, default_axis)
    return angle, axis
`

In which receives only one param.

## how to fix this
`@torch.jit.script
def quat_to_angle_axis(q: Tensor, w_last:bool) -> Tuple[Tensor, Tensor]:
    # type: (Tensor, bool) -> Tuple[Tensor, Tensor]
    # computes axis-angle representation from quaternion q
    # q must be normalized
    # ZL: could have issues. 
    min_theta = 1e-5
    qx, qy, qz, qw = 0, 1, 2, 3

    sin_theta = torch.sqrt(1 - q[..., qw] * q[..., qw])
    angle = 2 * torch.acos(q[..., qw])
    angle = normalize_angle(angle)
    sin_theta_expand = sin_theta.unsqueeze(-1)
    axis = q[..., qx:qw] / sin_theta_expand

    mask = torch.abs(sin_theta) > min_theta
    default_axis = torch.zeros_like(axis)
    default_axis[..., -1] = 1

    angle = torch.where(mask, angle, torch.zeros_like(angle))
    mask_expand = mask.unsqueeze(-1)
    axis = torch.where(mask_expand, axis, default_axis)
    return angle, axis`

**Comment by zhang705** [comment]
**what about**
def _reward_teleop_body_rotation_extend(self):
        rotation_diff = quat_to_angle_axis(self.dif_global_body_rot)[0]
        diff_body_rot_dist = (rotation_diff**2).mean(dim=-1)
        r_body_rot = torch.exp(-diff_body_rot_dist / self.config.rewards.reward_tracking_sigma.teleop_body_rot)
        return r_body_rot
